src/Microsoft.AspNetCore.Build/scripts/KoreBuild.ps1

```diff
@@ -1,28 +1,9 @@
 Write-Host -ForegroundColor Green "Starting KoreBuild 2.0 ..."
 
-if($env:KOREBUILD_COMPATIBILITY -eq "1") {
```

Comment:
Turns out there are enough things I want Universe to do differently for KoreBuild 2.0 repos that it's easier to do it on that end.